Jenkinsfile.trigger

```
@@ -211,7 +208,8 @@ node {
 
 					jq <<<"$currentJobsJson" '
 						# merge the two objects recursively, preferring data from "buildCompletionDataJson"
-						. * ( env.buildCompletionDataJson | fromjson )
+						# save firstTime if it isn't set yet
+						map_values(.firstTime //= .lastTime) * ( env.buildCompletionDataJson | fromjson )
```

Can't have `'` because we're inside `'` :smile:  also if we split this up the diff is slightly better and it's slightly more obvious that we're adding data _and_ munging data since they're separate

```suggestion
						. * ( env.buildCompletionDataJson | fromjson )
						# save firstTime if it is not set yet
						| map_values(.firstTime //= .lastTime)
```